_episodes/08-func.md

```diff
@@ -177,6 +177,28 @@ temperature in Kelvin was: 373.15
 ~~~
 {: .output}
 
+The variable `temp_kelvin`, being defined outside any function, is said to be [global]({{ page.root }}/reference.html#global-variable).
```

Comment:
This line of markdown should be split, to keep under the 100 character preference.